### DISCUSSION TITLE
[helm] supports azure blob storage external logs

### DISCUSSION THREAD

**Discussion by sivankumar86** [airbytehq/airbyte#36464] [Connector Ideas and Features]
### Topic

Azure Support

### Relevant information

At the moment, Airbyte support only s3 and gcs storage for logging however, we are using Azure hence, Azure blob storage support is required. 

https://azure.microsoft.com/en-au/products/storage/blobs

Workaround, using default option (minto)

**Comment by matheusnd** [comment]
The default option with Minio is not working for me. 
It is not clear if the helm-chart deploys a version of Minio or if we need to deploy our own Minio and provide access to the Helm.

[a comment by an automated account is not shown]
